fm4tlp/test.py

```diff
@@ -283,7 +289,7 @@ def main(_):
       structural_feature_dim=test_feature_dim,
   )
 
-  model_name = '_'.join(
```

Comment:
Please remove capitalization -- in general please adhere to https://google.github.io/styleguide/pyguide.html when making changes